app/views/posts/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1>List of All Posts Made by a Specific User</h1>
+<section>
+  <%= render "components/user-card", user: @user  %>
+  <%if @user.posts.any? %>
+    <ul class="posts-wrapper">
+      <% @user.posts.each do |post| %>
+        <li class="post-preview-card">
+          <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+            <%= render "components/post-preview-card", post: post %>
+
+            <%if post.comments.any? %>
+              <h4>Comments</h4>
+              <ul class="comments-wrapper">
+                <% post.comments[0, 5].each do |comment| %>
```

Comment:
- [ ] As mentioned in the project requirements, please use the last_five_comments` method you created in your Post model to render the last 3 posts.